src/JWT/Algorithms/RS256Algorithm.cs

```diff
@@ -43,28 +48,7 @@ public byte[] Sign(byte[] bytesToSign)
         /// <param name="signature">The signature to verify with</param>
         public bool Verify(byte[] bytesToSign, byte[] signature)
         {
-            var publicKey = GetPublicKey(_cert);
-            return publicKey.VerifyData(bytesToSign, "2.16.840.1.101.3.4.2.1", signature);
-        }
-
-        private static RSA GetPrivateKey(X509Certificate2 cert)
-        {
-#if NETSTANDARD1_3
-            return cert.GetRSAPrivateKey();
-#else
-            return (RSA)cert.PrivateKey;
-#endif
-        }
-
-        private static RSACryptoServiceProvider GetPublicKey(X509Certificate2 cert)
-        {
-            AsymmetricAlgorithm alg;
-#if NETSTANDARD1_3
-            alg = cert.GetRSAPublicKey();
-#else
-            alg = cert.PublicKey.Key;
-#endif
-            return (RSACryptoServiceProvider)alg;
+            return _publicKey.VerifyData(bytesToSign, "2.16.840.1.101.3.4.2.1", signature);

```

Comment:
Add a comment what this "magic number means"

Reply:
Not sure, it was there in the previous code.

Reply:
Before it was API/method calls only. Now it passes this OID. Simple comment what it means, or a url to the spec, would work!

Reply:
Did a google and the string matched the above. So added the comment.

Reply:
Definitely was passing in the OID previously btw https://github.com/jwt-dotnet/jwt/pull/195/files

Reply:
Right! Sorry, my bad. Was trying to review from my phone. Took my laptop. Much better now.

Reply:
BTW I don't think that Verify method is used anywhere. And the OID seems to be a non-RSA algorithm. So might be broken.

Reply:
`Verify()` was added as a request by a user so I'd keep it.